mtv_dl.py

```diff
@@ -929,6 +939,8 @@ def _move_to_user_target(self,
         if '{ext}' not in posix_target:
             posix_target += '{ext}'
 
+        # TODO: figure out what to do with the file name if season/episode are None
```

Comment:
I guess we just use an empty string or 0. Its not so easy to guess what people expect in this case.

Reply:
done